account_invoice_validate_send_email/README.rst

```diff
@@ -13,9 +13,9 @@ Account Invoice Validate Send Email
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Fmi7--custom-lightgray.png?logo=github
-    :target: https://github.com/qrtl/mi7-custom/tree/10.0/account_invoice_validate_send_email
-    :alt: qrtl/mi7-custom
+.. |badge3| image:: https://img.shields.io/badge/github-qtrl%2Fmi7--custom-lightgray.png?logo=github
+    :target: https://github.com/qtrl/mi7-custom/tree/1691-imp-separate-mail_servers-for-each-company/account_invoice_validate_send_email
+    :alt: qtrl/mi7-custom
```

Comment:
@tack-yabeQuartile Please fix the parameters in the README generation command - `qrtl` for the organization, `10.0` for the branch.